### PR TITLE
Move all the libraries to /vendor folder

### PR DESCRIPTION
**Pull request by LindaYu17**
Signed-off-by: Yu, Linda <linda.yu@intel.com>

### CODE REVIEW

File: va/Android.mk [hunk 1]
```diff
@@ -127,5 +129,6 @@ LOCAL_SHARED_LIBRARIES := libva
 
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libva-tpi
+LOCAL_PROPRIETARY_MODULE := true
```
Comment (xhaihao) [inline va/Android.mk]: libva-tpi is deprecated,  do you still use this library in Android?

File: va/Android.mk [hunk 2]
```diff
@@ -108,6 +109,7 @@ LOCAL_CFLAGS += \
 
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libva-egl
+LOCAL_PROPRIETARY_MODULE := true
```
Comment (xhaihao) [inline va/Android.mk]: libva provides a new mechanism to share buffer between libva and the 3rd library, are you sure this library still used on Android?

Comment (xhaihao) [inline va/Android.mk]: libva provides a new mechanism to share buffer between libva and the 3rd library, are you sure this library still used on Android?

Comment (LindaYu17) [inline va/Android.mk]: not used, let me remove it.

File: va/Android.mk (hunk 1 above)
Comment (LindaYu17) [inline va/Android.mk]: not used, let me remove it.

**Comment by xhaihao** [comment]
https://github.com/01org/libva/pull/47 contains the change of https://github.com/01org/libva/pull/46

**Pull request closed without merging**